Pop Alexia/problem 1/main.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+ int num;
+ printf("Introduce a number \n");
+ scanf("%d",&num);
+ if (num%3==0)
+    printf("The number is a multiple of 3 ");
+ else if (num%7==0)
+    printf("The number is a multiple of 7 ");
+ else if(num%3!=0|| num%7!=0)
```

Comment:
Correct! Great exercise for trying things out.

Please don't forget to format your code and put curly brackets 😄